### ISSUE TITLE
Installation Guide

### ISSUE THREAD

**Issue by AlexWHughes** [WWBN/AVideo-Encoder-Network#1]
Is there any more information on this? Upon installation I get the normal issues but once I fix the rewrite,etc I just get an blank page. Seems like this one has a manual config file. If so how do I set it set it correctly?

Tried feeding it MySQL and install details but the line " $global['systemRootPath'] = 'D:/Dropbox/Dropbox/htdocs/YouPHPTube-Encoder-Agregator/';" Stumped me

**Comment by AlexWHughes** [comment]
I worked out from old configs of YouPHPTube what the system root path is /var/www/YouPHP-Encoder-Network but I am still having no luck the moment I change the apache2.conf

**Comment by AlexWHughes** [comment]
I've even tried feeding it a normal Encoders database with no success.

**Comment by DanielnetoDotCom** [comment]
Sorry, we are not supporting this yet, it is not 100% done

**Comment by AlexWHughes** [comment]
Is there a way to aggregate or use more than one encoder currently?

**Comment by DanielnetoDotCom** [comment]
that is the way I was working to make this happening.
but for now, it is not 100% complete

**Comment by AlexWHughes** [comment]
So right now you can only use one encoder and process one file at a time?

**Comment by DanielnetoDotCom** [comment]
actually you can have multiple encoders, but you have to manually access each of them.
once the encode process is finished the encoder will send the video to the streamer site

**Comment by DanielnetoDotCom** [comment]
It is working already, you can see here. https://network.youphptube.com/

we dont have an installer for now, you will need to install the database manually and also add the encoders manually;

**Comment by AlexWHughes** [comment]
Do you have a database dump that we can use as a start point

**Comment by DanielnetoDotCom** [comment]
This?

https://github.com/DanielnetoDotCom/YouPHPTube/blob/master/install/database.sql

**Comment by AlexWHughes** [comment]
Installing it and attaching to YouPHP Database gives me the following error "SELECT * FROM streamers LIMIT 1\nError : ()" 

Despite the streamers table being present. Do I manually need to add encoders?

**Comment by DanielnetoDotCom** [comment]
@AlexWHughes you right, you have the right database, and yes you need to manually add the encoders

**Comment by DanielnetoDotCom** [comment]
This will be your streamer URL, user and pass. Just like it is on encoder

**Comment by raffeal** [comment]
i have install encoder network but notworking for
![ERROR](https://user-images.githubusercontent.com/47043451/54800831-c7ebee00-4c85-11e9-9c25-ed9642290d4c.png)
 my streamer

**Comment by raffeal** [comment]
my streamer is working fine on https://encoder.youphptube.com

**Comment by raffeal** [comment]
[22-Mar-2019 04:44:15 UTC] PHP Strict Standards:  Static function ObjectYPT::getTableName() should not be abstract in /home/pehchan/public_html/YouPHPTube-Encoder-Network/objects/Object.php on line 4
[22-Mar-2019 04:44:15 UTC] PHP Strict Standards:  Static function ObjectYPT::getSearchFieldsNames() should not be abstract in /home/pehchan/public_html/YouPHPTube-Encoder-Network/objects/Object.php on line 5

**Comment by DanielnetoDotCom** [comment]
@raffeal  what did you paste above is not an error

I guess you have a DNS problem, NOT a script problem

**Comment by raffeal** [comment]
i am reciveing these error from encoder network log

**Comment by bloomberg21** [comment]
I actually have it somewhat working but my server encoder is labeled as  OFFLINE and I cannot login no matter what I try.

The 2  public servers
- https://encoder.youphptube.com/
- https://encoder2.youphptube.com/

Show up just fine and I am logged in etc...

Any thoughts?
@DanielnetoDotCom

**Comment by DanielnetoDotCom** [comment]
I see

Just make sure the encoders and the network can see each other.

**Comment by bloomberg21** [comment]
Yup I have it running, removed the public servers and now have as a test, 4 of my encoders running and being shown as active.

**Comment by DanielnetoDotCom** [comment]
And is it working now?

**Comment by bloomberg21** [comment]
well... that is a trick question... LOL

Somewhat is my response.

I ran into a very odd issue...

Here is my set-up:

Encoder server running Network-encoder
4 folders in html/root named
1
2
3
4
each running the encoder sw. **<--- question here is do they each need their own DB and unique install or can they share a DB?**

Each config is related to each instance in 1-4

When I attempted to encode a file, all 4 instances were attempting to do so.

I know if we could work together on this we can make it much better experience for all.

**Comment by DanielnetoDotCom** [comment]
Each encoder needs his own DB

**Comment by faysal89** [comment]
after install encoder unable to go to encoder home page, 
automaticaly redirected to install directory

**Comment by DanielnetoDotCom** [comment]
This installation still with some bugs.

try to manually create the configuration.php file and give write permission to it. only after that, click in install.
